Resources/ProxyProvider/DIRACCAProxyProvider.py

```diff
@@ -365,19 +369,20 @@ def __parseDN(self, dn):
         dnInfoDict[f].append(v)
     return S_OK(dnInfoDict)
 
-  def __fillX509Name(self, field, value):
+  def __fillX509Name(self, field, values):
```

Comment:
How about that ?
 https://github.com/DIRACGrid/DIRAC/blob/42bb88ac3e2edb214afbdcaae83284a2e57519a1/Resources/ProxyProvider/DIRACCAProxyProvider.py#L440

Reply:
this will iterate over each char